tests/openwisp2/tests/user_batch_test.py

```diff
@@ -0,0 +1,317 @@
+import os
+
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.urls import reverse
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import Select, WebDriverWait
+
+from .test_selenium import SeleniumTestMixin
+
+current_script_path = os.path.dirname(os.path.abspath(__file__))
+
+
+class BasicTest(SeleniumTestMixin, StaticLiveServerTestCase):
+    def setUp(self):
+        self.admin = self._create_admin(
+            username=self.admin_username, password=self.admin_password
+        )
+
+    def test_batch_user_creation(self):
+        self.login()
+
+        """Test the batch user creation feature"""
+        self.web_driver.get(
+            f"{self.live_server_url}{reverse('admin:openwisp_users_organization_add')}"
+        )
+        self.web_driver.find_element(By.ID, "id_name").send_keys("Test Batch")
```

Comment:
Use `self.find_element` instead of `self.web_driver.find_element`. 

https://github.com/openwisp/openwisp-utils/blob/c16eaa441e694a68c1406eec9a6ecd640e486ff9/openwisp_utils/tests/selenium.py#L115-L128

Same applies for all the occurrences.